swift-signal-handle.rst

```diff
@@ -143,22 +154,25 @@ Full Example Template
                 sleep 300
 
                 # Assuming long running operation completed successfully, notify success signal
-                wc_notify --data-binary '{"status": "SUCCESS"}'
+                wc_notify --data-binary '{"status": "SUCCESS", "data": "Script execution succeeded"}'
+                
+                # Alternatively if operation fails a FAILURE with reason and data may be sent,
+                # notify failure signal example below
+                # wc_notify --data-binary '{"status": "FAILURE", "reason":"Operation failed due to xyz error", "data":"Script execution failed"}'
 
               params:
                 wc_notify: { get_attr: ['signal_handle', 'curl_cli'] }
 
-      wait_on_server:
-        type: OS::Heat::SwiftSignal
-        properties:
-          handle: {get_resource: signal_handle}
-          count: 1
-          timeout: 600
-
     outputs:
+      #Get the signal URL which contains all information passed to the signal handle
       signal_url:
         value: { get_attr: ['signal_handle', 'curl_cli'] }
         description: Swift signal URL
+      
```

Comment:
You need to make sure that the callout sections above match these changes to the resulting template.